### PR TITLE
Return selected contact participants in chat; extension point for adding top bar button in mini chat; fix old call button

### PR DESCRIPTION
**Pull request by NickEngineer**

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/ExoChatRoomDetail.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@
             </div>
           </div>
         </div>
-        <div v-if="contact.type ==='u' && contact.isEnabledUser === 'true'" class="callButtonContainerWrapper pull-left"></div>
```
Comment (pnedonosko) [inline application/src/main/webapp/vue-app/components/ExoChatRoomDetail.vue:27]: @NickEngineer why we need this?

Comment (pnedonosko) [reply application/src/main/webapp/vue-app/components/ExoChatRoomDetail.vue:27]: ok, it was a bug and this commit fixes it to work in group rooms well.

**Review by pnedonosko — APPROVED** [review]

File: application/src/main/webapp/vue-app/components/ExoChatRoomDetail.vue (hunk 1 above)
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatRoomDetail.vue:27]: Could you tell me more about the issue here ? this change will allow showing th ecall button for disabled users which will cause a regression ?

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: application/src/main/webapp/vue-app/components/ExoChatRoomDetail.vue (hunk 1 above)
Comment (NickEngineer) [reply application/src/main/webapp/vue-app/components/ExoChatRoomDetail.vue:27]: We had a problem with visualization. And we had to see this in spaces and chat rooms

Comment (NickEngineer) [reply application/src/main/webapp/vue-app/components/ExoChatRoomDetail.vue:27]: contact.isEnabledUser is null in chat rooms and type is t

Comment (NickEngineer) [reply application/src/main/webapp/vue-app/components/ExoChatRoomDetail.vue:27]: We fixed visualization logic. At the moment it's the same as for exo-chat-contact : v-if="contact.isEnabledUser === 'null' || contact.isEnabledUser === 'true'"

Please, check

**Review by ahamdi — APPROVED** [review]
LGTM :+1:

**Review by rdenarie — APPROVED** [review]

**Pull request merged**